app/src/main/java/com/github/se/cyrcle/ui/report/ParkingReportScreen.kt

```diff
@@ -0,0 +1,189 @@
+package com.github.se.cyrcle.ui.report
+
+import android.annotation.SuppressLint
+import android.widget.Toast
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.github.se.cyrcle.R
+import com.github.se.cyrcle.model.parking.ParkingReport
+import com.github.se.cyrcle.model.parking.ParkingReportReason
+import com.github.se.cyrcle.model.parking.ParkingViewModel
+import com.github.se.cyrcle.model.user.UserViewModel
+import com.github.se.cyrcle.ui.addParking.attributes.DESCRIPTION_MAX_LENGTH
+import com.github.se.cyrcle.ui.addParking.attributes.DESCRIPTION_MIN_LENGTH
+import com.github.se.cyrcle.ui.navigation.NavigationActions
+import com.github.se.cyrcle.ui.theme.Typography
+import com.github.se.cyrcle.ui.theme.atoms.BulletPoint
+import com.github.se.cyrcle.ui.theme.atoms.ConditionCheckingInputText
+import com.github.se.cyrcle.ui.theme.atoms.Text
+import com.github.se.cyrcle.ui.theme.disabledColor
+import com.github.se.cyrcle.ui.theme.molecules.EnumDropDown
+import com.github.se.cyrcle.ui.theme.molecules.TopAppBar
+
+const val MAX_CHARACTERS = 256
```

Comment:
I'd appreciate if you made your constant's name more explicit, and defined it in another file that is general to reports, not only to parking reports (e.g in the model package, where you defined the reports.)